daemon/graphdriver/rbd/README.md

```diff
@@ -0,0 +1,82 @@
+Ceph Graph Driver Documentation
+===============================
+
+# Known issues
+
+ - At the moment it is *not* possible to statically compile docker with
+   the ceph graph driver enabled because some static libraries are missing
+   (Ubuntu 14.04). Please use: ./hack/make.sh dynbinary
+
+# How to compile
+
+With Ubuntu 14.04:
+
+```bash
+# cd /root/go/src/github.com/docker
+# git clone https://github.com/docker/docker.git && cd docker
+# export GOPATH=/root/go/src/github.com/docker/docker/vendor/:/root/go
+
+# hack/make.sh dynbinary
+# WARNING! I don't seem to be running in the Docker container.
+# The result of this command might be an incorrect build, and will not be
+#   officially supported.
+#
+# Try this instead: make all
+#
+
+bundles/1.8.0-dev already exists. Removing.
+
+---> Making bundle: dynbinary (in bundles/1.8.0-dev/dynbinary)
+Created binary: bundles/1.8.0-dev/dynbinary/dockerinit-1.8.0-dev
+Building: bundles/1.8.0-dev/dynbinary/docker-1.8.0-dev
+Created binary: bundles/1.8.0-dev/dynbinary/docker-1.8.0-dev
+```
+# How to use
+
+## install ceph cluster
+TODO:
```

Comment:
Perhaps you get suggest to use the ceph/demo container for testing. https://github.com/ceph/ceph-docker/tree/master/demo